StyleCop.Analyzers/StyleCopTester/Program.cs

```diff
@@ -39,6 +39,10 @@ private static void Main(string[] args)
                     cts.Cancel();
                 };
 
+            // The following allows continuations to be processed by a single thread.
```

Comment:
:thought_balloon: We use `ConfigureAwait(false)` pretty much everywhere, so the comment about using a single thread is misleading at best, but probably incorrect.